.github/workflows/pinot_dbapi_test.yml

```diff
@@ -40,7 +40,7 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
Shall we separate jobs from `pinot-dbapi-test` to `pinot-dbapi-unit-test` and `pinot-dbapi-integration-test`

Reply:
Yeah, good point, I'll do that now.

Reply:
Done!